web/static/js/components/surveys/SurveyWizardRespondentsStep.jsx

```diff
@@ -192,7 +194,7 @@ class SurveyWizardRespondentsStep extends Component {
 
     const removeRespondentsLink = (groupId) => {
       if (!surveyStarted && !uploading) {
-        return <a href='#' className='modal-trigger' style={{marginRight: '0'}} onClick={() => this.openRemoveRespondentModal(this.refs.removeRespondents, groupId)}>{t('Remove respondents')}</a>
+        return <a href='#' className='modal-trigger' style={{marginRight: '0'}} onClick={(e) => this.openRemoveRespondentModal(e, this.refs.removeRespondents, groupId)}>{t('Remove respondents')}</a>
```

Comment:
```suggestion
        return <a href='#' className='modal-trigger' style={{marginRight: '0'}} onClick={e => this.openRemoveRespondentModal(e, this.refs.removeRespondents, groupId)}>{t('Remove respondents')}</a>
```

FYI parens on single args are not needed. No need to change.